go.mod

```diff
@@ -26,7 +26,7 @@ require (
 	github.com/containerd/typeurl v1.0.2
 	github.com/coreos/go-systemd/v22 v22.4.0
 	github.com/docker/cli v23.0.0-rc.1+incompatible
-	github.com/docker/distribution v2.8.1+incompatible
+	github.com/docker/distribution v2.8.2-beta.1+incompatible
```

Comment:
This should be v2.8.2, not the `beta.1`

I should also add that both CVE-2022-28391 and CVE-2023-2253 only affect the registry _binary_, not the library code used in this repository (https://github.com/distribution/distribution/releases/tag/v2.8.2), so scanners flagging this as a vulnerability in BuildKit are effectively reporting a false positive.